modules/button/css/README.md

```diff
@@ -237,17 +240,20 @@ Apply primary styling to the button by adding `.wdc-btn-primary` to the buttons
   <button class="wdc-btn wdc-btn-split-text">
     Split Button
   </button>
-  <button class="wdc-btn wdc-btn-split-icon" />
+  <button class="wdc-btn wdc-btn-split-icon" aria-haspopup="true" />
```

Comment:
Do we use Split Buttons anywhere? I didn't see any in the React kit. Also, this button should have an `aria-label` or `aria-labelledby` or maybe both:

```html
  <button class="wdc-btn wdc-btn-split-text" id="split-button">
    Split Button
  </button>
  <button class="wdc-btn wdc-btn-split-icon" aria-haspopup="true" aria-label="More Options" aria-labelledby="split-button" />
```